ChristmasLights/java/src/main/java/christmaslights/LightService.java

```diff
@@ -0,0 +1,8 @@
+package christmaslights;
+
+public interface LightService {
+    void on(int startX,int startY, int endX,int endY);
+    void off(int startX,int startY, int endX,int endY);
+    void toggle(int startX,int startY, int endX,int endY);
+    int countLights();
```

Comment:
This violates interface seggregation principle... create seperate interfaces for each